benchmark/src/bench_client.rs

```diff
@@ -127,7 +111,7 @@ impl BenchClient {
     ///
     /// If `XlineClient` or `EtcdClient` failed to send request
     #[inline]
-    #[cfg(test)]
+    #[allow(unused)]
     pub(crate) async fn get(
```

Comment:
This method only be used in tests. I think `cfg(test)` is better than `allow(unused)`